client/src/components/composables/media-player.ts

```diff
@@ -176,3 +176,8 @@ export function usePlaybackRate() {
 		availablePlaybackRates,
 	};
 }
+
+export enum MediaPlayerError {
+	none = 0,
+	network = 1,
+}
```

Comment:
Would prefer this to not be a ts enum. Should be a union instead.

I know this isn't consistent with the rest of the codebase, but I've changed my opinion on ts enums since then.